metabase/metabase
# feat(sdk): Allow to hide columns in CollectionBrowser

Closes https://github.com/metabase/metabase/issues/49232

### Description

This exposes a prop to specify visible columns for CollectionBrowser.

### How to verify

1. Run SDK storybook
2. Check CollectionBrowser > Without Some Columns story

### Demo

https://github.com/user-attachments/assets/ee19aefc-0a0b-4f79-be41-f3f24b9b27fa

### Checklist
